src/js/events/chromecastreceiver-events.js

```diff
@@ -0,0 +1,44 @@
+const ChromecastReceiverEvents = [
+    'messageReceived',
+    'requestLoad',
+    'setCredentials',
+    'editAudioTrack',
+    'editTracksInfo',
+    'loadByEntity',
+    'userAction',
+    'displayStatus',
+    'customCommand',
+    'focusState',
+    'queueLoad',
+    'queueInsert',
+    'queueUpdate',
+    'queueRemove',
+    'queueReorder',
+    'queueGetItemRange',
+    'queueGetItems',
+    'queueGetItemsIndex',
+    'castError',
+    'playerLoading',
+    'playerPreloading',
+    'playerPreloadingCancelled',
+    'canPlay',
```

Comment:
Shouldn't there be something in the event name to indicate this is exclusive to ChromeCast? I see several names that are ambiguous and could easily apply to several other plugins.